backend/mcp_server/views.py

```
@@ -0,0 +1,95 @@
+            return None
+
+        # Pin the organization before touching org-scoped managers; the
+        # deployment lookup below filters on it.
+        DeploymentHelper.validate_parameters(
+            request, api_name=api_name, org_name=org_name
+        )
+
+        api_deployment = DeploymentHelper.get_deployment_by_api_name(api_name=api_name)
+        try:
+            DeploymentHelper.validate_api(api_deployment=api_deployment, api_key=api_key)
+        except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)